### ISSUE TITLE
Show when a flag was raised (and handled) in the flag queue

### ISSUE THREAD

**Issue by cellio** [codidact/qpixel#1307]
Sometimes when I see a flag while moderating, I would like to know when the flag was raised.  Sometimes it's important to know if a flag came before or after an edit, or where it fell in relation to a comment thread the flagger is involved in, and sometimes as an admin I'd like to know if local mods have had a chance to see it yet. The flag queue shows me the post, the flag, and the flagger, but not the timestamp of the flag.  Can we add this information, perhaps after the flagger's name in the entry?

If we do this, for handled flags could we also add the timestamp for when it was handled?  (For consistency, and to allow mod teams to understand what happened in those rare cases where that matters.)

The other place flags show up is in the per-user flag page.  I don't have a strong opinion about whether to show it there. I don't think there's a reason to withhold it, and I *think* the underlying code is the same so it's probably easier to show it, but either's fine with me.

I took a look at first the controller and then the model, but I couldn't figure out how to get timestamp information.

**Comment by ArtOfCode-** [comment]
Hint - Most models have `created_at` and `updated_at` fields that you can get timestamps from.

**Comment by cellio** [comment]
How odd.  I see from `app/views/flags/_flag.html.erb` that flags have `handled_by`, and I took a guess that there would be `handled_at` and tried it and that works -- *but* I see no such thing in `app/models/flag.rb`.  Where does that come from?  If I hadn't had the clue from `handled_by`, where should I have have looked to find out that `handled_at` exists?

I've got this working now, and arguably I should ask this in chat, but if I ask here then other beginners and future-me might be a little more likely to learn from it.
